docs/everest.md

```diff
@@ -0,0 +1,12 @@
+# Install Percona Distribution for PostgreSQL cluster using Everest
+
+[Percona Everest :octicons-link-external-16:](https://docs.percona.com/everest/) is an open source cloud-native database platform that helps developers deploy code faster, scale deployments rapidly, and reduce database administration overhead while regaining control over their data, database configuration, and DBaaS costs.
+
+It automates day-one and day-two database operations for Postgres, MySQL, and MongoDB databases within Kubernetes clusters.
```

Comment:
```suggestion
It automates day-one and day-two database operations for open source databases on Kubernetes clusters.
```

Reply:
the idea is that we should not mention mysql, mongo in PG doc. Also Everest will grow to support other databases. So let's simplify.